conf/messages

```diff
@@ -869,8 +869,14 @@ uploadAuthority.duty = duty
 uploadAuthority.both = import VAT and duty
 uploadAuthority.p2 = The proof needs to be dated and signed by the owner of the deferment account. The date must be after the date of the original import declaration.
 uploadAuthority.fileSizeTypeHeading = File size and type
-uploadAuthority.fileSizeTypeText = The file must be 6MB or smaller and either be a PDF (.pdf) or a text (.txt) file.
+uploadAuthority.typesOfFile = Types of file you can upload
```

Comment:
some of these are duplicated in 3 places
`uploadAuthority.typesOfFile`
`uploadFile.typesOfFile`
`bulkUploadFile.typesOfFile`
should we reuse them?
would we ever want to allow different types of documents in different forms?